zigbeeeurotronic/zigbeeeurotronic.pro

```diff
@@ -0,0 +1,14 @@
+include(../plugins.pri)
+
+PKGCONFIG += nymea-zigbee
+
+SOURCES += \
+    ../common/zigbeeintegrationplugin.cpp \
```

Comment:
Since this is the second place where we use common base classes for multiple integrations (also in nymea-plugins-modbus), would it make sense to provide the base class as lib? Otherwise we compile the same class x times into binaries but they are all the same. The footprint would be a bit less I think...not much, but a bit.

Reply:
Yes, a lib would make sense... It's reasonably tiny and used in 2 places atm so it probably doesn't change anything in practice yet, but yes, if this grows or used in more plugins, it'll be turned into a lib.